### ISSUE TITLE
[LB-Next]  Assess amount of work to convert existing LB3 connector to TypeScript

### ISSUE THREAD

**Issue by dhmlau** [strongloop/loopback-next#421]
Assess how much work to convert existing connectors to TypeScript.  Similar to the MySQL connector interface done in https://github.com/strongloop/loopback-next-example/pull/51.
It includes functions like `search`, `findById`, etc. 

## Acceptance Criteria
- Build MySQL connector (use this as an example, seems like there're existing code we can reference)
- Create (reuse) tests
- List out the work items need to be done for this "conversion"

_Note: feel free to edit accordingly.  Possibly needs grooming._

_Coming from discussion with @raymondfeng_

## BLOCKED BY

 - https://github.com/strongloop/loopback-next/issues/537

**Comment by bajtos** [comment]
I think rewrite of connectors to TypeScript is pretty low priority compared to all other GA (and postGA) related tasks we have in front of us. I am closing this issue as not relevant for now.
